### PR TITLE
Remenska

### PR DESCRIPTION
**Pull request by remenska**
Initial

### CODE REVIEW

**Comment by blootsvoets** [comment]
For root, please add organizations and make the urls full urls.

**Comment by remenska** [comment]
which urls are relative?

**Comment by blootsvoets** [comment]
What do you mean? At least all URLs should not end in `/`

File: _project/pandas-root.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- ROOT binary format
+- pandas DataFrames
+contactPerson: http://software.esciencecenter.nl/person/d.remenska
+coordinator: http://software.esciencecenter.nl/person/j.maassen
+engineer:
+- http: http://software.esciencecenter.nl/person/d.remenska
+principalInvestigator:
+  - name: Dr. Christopher Tunnell
+    affiliation:
+    - http:
```
Comment (blootsvoets) [inline _project/pandas-root.md]: Add affiliation

File: _software/root-conda-recipes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Physics & Beyond
+expertise:
+- Distributed Computing
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/d.remenska
+owner: 
+- http://software.esciencecenter.nl/person/d.remenska
+contributor:
+- http://software.esciencecenter.nl/person/d.remenska
+- http://software.esciencecenter.nl/person/s.verhoeven
```
Comment (blootsvoets) [inline _software/root-conda-recipes.md]: Add s.verhoeven contributorOf value

File: _project/pandas-root.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+expertise:
+  - Linked Data
+  - High Performance Computing
+dataMagnitude: TB
+dataFormat: 
+- JSON
+- ROOT binary format
+- pandas DataFrames
+contactPerson: http://software.esciencecenter.nl/person/d.remenska
+coordinator: http://software.esciencecenter.nl/person/j.maassen
```
Comment (blootsvoets) [inline _project/pandas-root.md]: Add coordinatorOf to Jason

File: _project/pandas-root.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+engineer:
+- http: http://software.esciencecenter.nl/person/d.remenska
+principalInvestigator:
+  - name: Dr. Christopher Tunnell
+    affiliation:
+    - http:
+    website: http://www.nikhef.nl/~decowski/Group.html
+involvedOrganization:
+- http://software.esciencecenter.nl/organization/nlesc
```
Comment (blootsvoets) [inline _project/pandas-root.md]: Add involvedIn to nlesc

File: _project/pandas-root.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  - High Performance Computing
+dataMagnitude: TB
+dataFormat: 
+- JSON
+- ROOT binary format
+- pandas DataFrames
+contactPerson: http://software.esciencecenter.nl/person/d.remenska
+coordinator: http://software.esciencecenter.nl/person/j.maassen
+engineer:
+- http: http://software.esciencecenter.nl/person/d.remenska
```
Comment (blootsvoets) [inline _project/pandas-root.md]: add remenska engineerOf

File: _software/root-conda-recipes.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+user:
+- http://software.esciencecenter.nl/organization/nlesc
```
Comment (blootsvoets) [inline _software/root-conda-recipes.md]: Add uses to NLeSC

File: _software/root-conda-recipes.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+usedIn:
+- http://software.esciencecenter.nl/project/pandas-root
+status: active
+dependency:
+- ROOT
+- root-numpy
+- rootpy
+dependencyOf:
+technologyTag:
+- Anaconda, pandas DataFrame, Computing model
```
Comment (blootsvoets) [inline _software/root-conda-recipes.md]: add as array elements

**Pull request merged**